### PR TITLE
Fixing and Refactoring

### PR DESCRIPTION
**Pull request by Linzeur**
Refactor code about Feature #4 
Fix code because of refactoring.

### CODE REVIEW

File: controller/entry_methods.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require 'json'
+require "date"
```
Comment (deyviconde) [inline controller/entry_methods.rb]: Why ?

File: controller/entry_methods.rb [hunk 2]
```diff
@@ -9,20 +10,22 @@ def read_data
   data
 end
 
-def save_data(entry)
+def save_data(id, entry)
+  data[id] = entry
+  File.open($path_data, "w+") do |file|
+    file.write data.to_json
```
Comment (deyviconde) [inline controller/entry_methods.rb]: ok !

File: controller/entry_methods.rb [hunk 3]
```diff
@@ -9,20 +10,22 @@ def read_data
...
+  end
+end
+
+def add_data(entry)
   data_file = read_data
   identifier = Time.now.getutc.to_i
+  entry["datetime"] = DateTime.parse(entry["datetime"]).strftime("%Y%m%d%H%M%S")
```
Comment (deyviconde) [inline controller/entry_methods.rb]: This fix our datetime problem, cool !

File: public/css/style.css [hunk 4]
```diff
@@ -71,7 +70,8 @@ textarea {
   object-fit: cover;
   padding: 15px;
   border: 1px solid black;
-=======
+}
+
 .overflow {
   width: 16em;
```
Comment (deyviconde) [inline public/css/style.css]: What's the difference between em,px, % ?

File: public/css/style.css [hunk 5]
```diff
@@ -92,5 +92,4 @@ textarea {
 .menu:hover {
   text-decoration: none;
```
Comment (deyviconde) [inline public/css/style.css]: Why none? xD

File: route.rb [hunk 6]
```diff
@@ -43,7 +43,7 @@
   @data = list_daily
   erb :list
   erb :add_photo
-  erb :daily
+  erb :entry
```
Comment (deyviconde) [inline route.rb]: Oh, finally!!! This fixs our headache !

File: controller/entry_methods.rb (hunk 1 above)
Comment (Linzeur) [reply controller/entry_methods.rb]: It's for using methods to parse string to DateTime

File: public/css/style.css (hunk 4 above)
Comment (Linzeur) [reply public/css/style.css]: They are type of size a visual level.

File: route.rb [hunk 7]
```diff
@@ -27,13 +27,13 @@
 post '/' do
   inputs = Hash.new
   inputs["title"] = params["title"]
-  inputs["datetime"] = params["date"] + params["time"]
+  inputs["datetime"] = params["date"] + " " + params["time"]
```
Comment (liannivin) [inline route.rb]: Good.

**Pull request merged**